mongodbatlas/organization_invitations.go

```diff
@@ -102,12 +102,12 @@ func (s *OrganizationsServiceOp) Invitation(ctx context.Context, orgID, invitati
 // InviteUser invites one user to the Atlas organization that you specify.
 //
 // See more: https://docs.atlas.mongodb.com/reference/api/organization-create-one-invitation/
-func (s *OrganizationsServiceOp) InviteUser(ctx context.Context, invitation *Invitation) (*Invitation, *Response, error) {
-	if invitation.OrgID == "" {
```

Comment:
guessing this must be blank when POSTing, right? if so can you please update the description of the PR to include the why of the change?

Reply:
got the same issue with the update endpoints

Reply:
updated